tsconfig.base.json

```diff
@@ -15,6 +15,7 @@
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "paths": {
+      "@angular-challenges/angular/di": ["libs/angular/di/src/index.ts"],
```

Comment:
?

Reply:
@tomalaforge i was using nx schematics for generating pipes as well, i might have turned on a flag that modifed this tsconfig.base.json , i will check this out